src/Microsoft.AspNet.Mvc.Core/ActionResults/ViewExecutor.cs

```diff
@@ -14,7 +17,10 @@ namespace Microsoft.AspNet.Mvc
     public static class ViewExecutor
     {
         private const int BufferSize = 1024;
-        private const string ContentType = "text/html; charset=utf-8";
+        private static readonly MediaTypeHeaderValue DefaultContentType = new MediaTypeHeaderValue("text/html")
+        {
+            Encoding = Encoding.UTF8
```

Comment:
Same comments as earlier : Use UTF8 Without BOM and use the static helper we have.